### PR TITLE
travis: reconcile with homebrew/science

### PR DESCRIPTION
**Pull request by jonchang**
See https://github.com/Homebrew/homebrew-science/pull/5501

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+dist: trusty
+
+addons:
+  apt_packages:
+    - csh
+    - cython
+    - python-astropy
+    - python-nose
+    - python-pip
+    - python-sphinx
```
Comment (sjackman) [inline .travis.yml]: Please remove the `addons`. On Travis I prefer to have as few host dependencies installed as possible.

File: .travis.yml [hunk 2]
```diff
@@ -39,22 +47,28 @@ before_install:
...
   - export HOMEBREW_DEVELOPER=1
   - export HOMEBREW_NO_AUTO_UPDATE=1
   - export HOMEBREW_VERBOSE=1 HOMEBREW_VERBOSE_USING_DOTS=1
+  - env | grep TRAVIS | tee /tmp/travis.env
   - ulimit -n 1024
   - brew tap homebrew/science
   - brew tap linuxbrew/xorg
 
+install:
+  - pip install pysam
```
Comment (sjackman) [inline .travis.yml]: Ditto, please remove this chunk.

File: .travis.yml [hunk 3]
```diff
@@ -39,22 +47,28 @@ before_install:
...
+
 script:
+  - brew tap linuxbrew/xorg
```
Comment (sjackman) [inline .travis.yml]: This line doesn't appear to be needed, because it was working before. Please remove it.

**Comment by sjackman** [comment]
:shipit: Thanks, Jon!

**Comment by jonchang** [comment]
Squashed and merged. Hopefully this makes merging changes to this copy and the one in science much easier.

**Pull request merged**